src/lxc/confile.c

```diff
@@ -5772,6 +5878,60 @@ static int get_config_net_veth_pair(const char *key, char *retv, int inlen,
 	return fulllen;
 }
 
+static int get_config_net_veth_vlan_id(const char *key, char *retv, int inlen,
+				  struct lxc_conf *c, void *data)
+{
+	int len;
+	int fulllen = 0;
+	struct lxc_netdev *netdev = data;
+
+	if (!retv)
+		inlen = 0;
+	else
+		memset(retv, 0, inlen);
+
+	if (!netdev)
+		return -1;
+
+	if (netdev->type != LXC_NET_VETH)
+		return 0;
```

Comment:
This can probably be reversed, i.e.:
```c
if (!netdev)
        return ret_errno(ENOENT);

if (netdev->type != LXC_NET_VETH)
        return 0;

if (!retv)
.
.
.
```